README.md

```diff
@@ -4,6 +4,35 @@
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/silverstripe/cwp-core/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/silverstripe/cwp-core/?branch=master)
 [![codecov](https://codecov.io/gh/silverstripe/cwp-core/branch/master/graph/badge.svg)](https://codecov.io/gh/silverstripe/cwp-core)
 
+## About this module
+This module includes core configuration that is required for Common Web Platform sites to function correctly.
+
+### Configuration
+There are some settings that can be modified in this module, depending on requirements. These are listed below.
+
+#### XSS Protection
+By default, CWP sites instruct newer browsers to protect against cross-site scripting (XSS) attacks. This is done using an HTTP header (X-XSS-Protection). More information on this header can be found on the [Mozilla Developer Network](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection) site. To disable this feature, add the following to your YML configuration:
+```
+CWP\Core\Control\InitialisationMiddleware:
+  xss_protection_disabled: true
```

Comment:
Not a big fan of double negatives - it should be `xss_protection_enabled: false`